src/CommunityToolkit.Maui/Views/Popup/Popup.ios.macos.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using CoreGraphics;
+using Foundation;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Views.Popup;
+
+class Popup
+{
+	public Popup()
+	{
+		PopupView = new PopupView();
+
+		PopupView.ParentView?.AddSubview(PopupView);
+		PopupView.ParentView?.BringSubviewToFront(PopupView);
+	}
+
+	NSTimer? timer;
+
+	public TimeSpan Duration { get; set; }
+
+	public UIView? Anchor { get; set; }
+
+	public CGRect CornerRadius { get; set; }
+
+	public UIColor BackgroundColor { get; set; } = UIColor.Gray;
```

Comment:
Let's move`BackgroundColor` to `PopupView`.

I'd prefer for `Popup` to contain the functionality (e.g. Anchor, Timer, etc.) and `PopupView` to contain the UI elements .